waveanalysis/__main__.py

```diff
@@ -1,41 +1,37 @@
-import os                                       
-import sys 
-import timeit
-import datetime
-import numpy as np
-import pandas as pd
-import seaborn as sns
-from tqdm import tqdm
-import matplotlib.pyplot as plt
-from waveanalysis.waveanalysismods.customgui import BaseGUI, RollingGUI
-from waveanalysis.waveanalysismods.processor import TotalSignalProcessor, RollingSignalProcessor
-
-np.seterr(divide='ignore', invalid='ignore')
+import housekeeping.housekeeping_functions as hf
+from custom_gui import BaseGUI, RollingGUI, KymographGUI
+from data_workflows.combined_workflow import combined_workflow
+from data_workflows.rolling_workflow import rolling_workflow
```

Comment:
these  to be `import waveanalysis.housekeeping...` or `from waveanalysis...` because (for example) `housekeeping` won't be importable unless the user clones the repo and runs from within `waveanalys`